A34-weighted-round-robin.md

```diff
@@ -106,6 +104,4 @@ The reasons to introduce a new algorithm instead of using existing `weighted_tar
 N/A
 
 ## Open issues (if applicable)
-
-- Replace `round_robin` with `weighted_round_robin`?
-- Following first issue, if not, do we want to use `weighted_round_robin` as default lb_policy for eDS response?
\ No newline at end of file
+- Do we need a way to opt out WRR even per endpoint weight is assigned by ClusterLoadAssignment?
```

Comment:
If the new policy behaves exactly like round_robin when no endpoint weights are provided by the xDS server, then we probably won't need an opt-out mechanism.  But we'll need to verify that that's the case.

Reply:
based on
```
 Endpoints do not have `load_balancing_weight` is assigned to 1 (the smallest possible weight). 
```

yes, it will behave same as simple RR.